webapi/src/test/scala/org/knora/webapi/models/OntologyModels.scala

```diff
@@ -81,6 +97,12 @@ object PropertyValueType {
   case object IntValue extends PropertyValueType {
     val value = "knora-api:IntValue"
   }
+  case object LinkValue extends PropertyValueType {
+    val value = "knora-api:LinkValue"
+  }
+  case object Resource extends PropertyValueType {
```

Comment:
What about other values e.g. `BooleanValue`, `TimeValue` etc.?

Reply:
can be added as needed, I didn't want to write out the entire knora-base ontology again - especially since I don't know if we will hold on to these models